### PR TITLE
Audiotags2 -> Audiotags

### PR DESCRIPTION
**Pull request by pinkforest**
* Merging back fork
* TBD: Have to revert some commits

### CODE REVIEW

**Comment by martpie** [comment]
👀

**Comment by pinkforest** [comment]
@martpie can you have a look how the changes look pls? :) :unicorn:

File: LICENSE [hunk 1]
```diff
@@ -1,5 +1,6 @@
 MIT License
 
+Copyright (c) 2020 Tianyi Shi
 Copyright (c) 2022 Pierre de la Martinière
```
Comment (martpie) [inline LICENSE:4]: Feel free to remove me, that's all fine

**Review by martpie — APPROVED** [review]
LGTM modulo not passing CI, happy to see the project live on!

File: LICENSE (hunk 1 above)
Comment (pinkforest) [reply LICENSE:4]: it's a party y'know :)

**Pull request merged**